### PR TITLE
Feature/blogs

### PR DESCRIPTION
**Pull request by daquinohd**

### CODE REVIEW

**Comment by daquinohd** [comment]
@lburack @VictoriaSunNIH - the pipeline for the Blog epic has been deployed to http://ncigovcdode69.prod.acquia-sites.com:

The following are ready for testing and have been marked for IA review:
#351 (right rail - archive)
#450 (recommended content)
#861 (remove site sections from blog posts)
#1057 (number of posts to display bug)
#1071 (older/newer translation issue bug)
#1184 (right rail - featured content)
#1185 (archive landing pages)
#1186 (right rail - filter categories)
#1205 (Blog FEQ changes)

Existing issues:
* Blog Category landing pages are not in content yet
* RSS is not included
* Disqus/HHS issue is ongoing
* Translation behavior is still wonky (I believe this is a sitewide issue)

**Comment by daquinohd** [comment]
@lburack @VictoriaSunNIH - these tickets are ready for testing on *http://ncigovcdode69.prod.acquia-sites.com*

#351
#357
#450
#861
#1071
#1186
#1184
#1185
#1241

**Comment by lburack** [comment]
@daquinohd @alifr — @VictoriaSunNIH  and I are creating a new Blog Bugs punchlist which we will finish tomorrow, but here's an update on the tickets mentioned above:
- #351 - Blog Archive passed IA review
- #357 - Blog Commenting passed IA review
- #450 - there are issues with Recommended Content (these are included in our punchlist right now)
  - in the Drupal form, the default should be to Add Internal Promo Card, not an external one
  - 4x3 image crop should display
  - a piece of content is added to a blog post as Recommended Content. If the card/short title of that piece of content is then updated, I'm not seeing that change be reflected on the blog post until I push it through the workflow. The title used for Recommended Content should update on its own. 
- #861 - Remove Site Sections and Autogenerate URL path passed IA review, although I left a note about Spanish URLs here.
#1071 -  _< Artículo anterior_ and _Artículo siguiente >_ are now correctly appearing on Spanish blog posts. There are other Spanish issues though. See #1071 for my comments. 
#1186 - Filter Blog Categories by Series — @daquinohd is this for when you create a blog post, to only show Categories available to the series the post is associated with? If so, this passes IA review.
#1184 - Remove hardcoded Featured content from right rail passed IA review
#1185 - Create blog archive landing page passed IA review

**Comment by daquinohd** [comment]
> #861 - Remove Site Sections and Autogenerate URL path passed IA review, although I left a note about Spanish URLs here.
> #1071 -  _< Artículo anterior_ and _Artículo siguiente >_ are now correctly appearing on Spanish blog posts. There are other Spanish issues though. See #1071 for my comments.
@lburack - these seem to be part of the sitewide translation issues; will continue to work on it.
> #1186 - Filter Blog Categories by Series — @daquinohd is this for when you create a blog post, to only show Categories available to the series the post is associated with? If so, this passes IA review.
That is correct. Before, the categories block was showing all categories.

**Comment by VictoriaSunNIH** [comment]
Bug Tickets have been opened:
https://github.com/NCIOCPL/cgov-digital-platform/issues/1306
https://github.com/NCIOCPL/cgov-digital-platform/issues/1307
https://github.com/NCIOCPL/cgov-digital-platform/issues/1308

**Comment by alifr** [comment]
Sorry. I toggled the tags on the wrong ticket.

**Comment by VictoriaSunNIH** [comment]
@lburack , @amydugan Documenting that the only items left for this are: breadcrumbs not appearing correctly, the incorrect crop showing for recommended content, and the list description field.  @daquinohd - let us know what should be split off, and when the last few items might be done so we can call this passed!

**Comment by daquinohd** [comment]
@VictoriaSunNIH @lburack @amydugan
- Let's split the breadcrumb off as a new ticket. Since Blog Posts don't technically have a site section, we'll need to update the breadcrumb plugin, which has implications for the rest of the site.
- I'm still actively working on the recommended content crop; if I can't get it fixed by tonight, I'll open a separate ticket. This also has a workaround if we need it.
- I closed the List Description field issue (#1369) since this will be fixed as part of a shared field ticket.

File: docroot/profiles/custom/cgov_site/modules/custom/cgov_blog/src/Plugin/Block/CgovDisqus.php [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    if ($post_node = $this->getCurrEntity()) {
+      $series_nid = $post_node->get('field_blog_series')->target_id;
+      $series_node = $this->getNodeStorage()->load($series_nid);
+    }
+
+    // If 'Allow Comments' is selected, output the Disqus snippet data.
+    if ($series_node && $series_node->get('field_allow_comments')->value) {
+
+      // TODO: GET THE CORRECT ENVIRONMENT VARIABLE FOR PROD.
+      $tier = 'dev';
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/modules/custom/cgov_blog/src/Plugin/Block/CgovDisqus.php]: This needs to get cleaned up...

Comment (daquinohd) [reply docroot/profiles/custom/cgov_site/modules/custom/cgov_blog/src/Plugin/Block/CgovDisqus.php]: @bryanpnih This is cleaned up. I'll create a more general-use isProd() method as part of the DTM tag story.

**Pull request merged**